docs/Privacy/Configuring-Privacy.md

```diff
@@ -0,0 +1,168 @@
+
+## 4. Create Orion Configuration Files
+   
+In the `Node-1/Orion` directory, create a file called `orion.conf` and add the following properties:
+   
+```bash
+nodeurl = "http://127.0.0.1:8080/"
+nodeport = 8080
+clienturl = "http://127.0.0.1:8888/"
+clientport = 8888
+publickeys = ["nodeKey.pub"]
+privatekeys = ["nodeKey.key"]
+passwords = "passwordFile"
+tls = "off"
+```
+
+!!! important 
+    In production systems, only specify [`tls`](http://docs.orion.pegasys.tech/en/latest/Configuring-Orion/Configuration-File/#tls)
+    is `off` if another transport security mechanism such as WireGuard is in place. 
+    
+In the `Node-2/Orion` and `Node-3/Orion` directories, create `orion.conf` files specifying: 
+
+* Different ports
+* Node-1 Orion node as the bootnode (specified by [`othernodes`](http://docs.orion.pegasys.tech/en/latest/Configuring-Orion/Configuration-File/)): 
+
+```bash tab="Node-2"
+nodeurl = "http://127.0.0.1:8081/"
+nodeport = 8081
+clienturl = "http://127.0.0.1:8889/"
+clientport = 8889
+publickeys = ["nodeKey.pub"]
+privatekeys = ["nodeKey.key"]
+passwords = "passwordFile"
+othernodes = ["http://127.0.0.1:8080/"]
+tls = "off"
+```
+
+```bash tab="Node-3"
+nodeurl = "http://127.0.0.1:8082/"
+nodeport = 8082
+clienturl = "http://127.0.0.1:8890/"
+clientport = 8890
+publickeys = ["nodeKey.pub"]
+privatekeys = ["nodeKey.key"]
+passwords = "passwordFile"
+othernodes = ["http://127.0.0.1:8080/"]
+tls = "off"
+```
+   
+## 5. Start Orion Nodes
+   
+In each `Orion` directory, start Orion specifying the [configuration file](#3-create-a-configuration-file) created in the previous step:
+   
+```
+orion orion.conf
+``` 
+
+## 6. Start Pantheon Node-1
+
+In the `Node-1` directory, start Pantheon Node-1:
+
+```bash tab="MacOS"
+pantheon --data-path=data --genesis-file=../ibftGenesis.json --rpc-http-enabled --rpc-http-api=ETH,NET,IBFT,EEA --host-whitelist=* --rpc-http-cors-origins="all" --privacy-enabled --privacy-url=http://127.0.0.1:8888 --privacy-public-key-file=Orion/nodeKey.pub    
+```
+
+```bash tab="Windows"
+pantheon --data-path=data --genesis-file=..\ibftGenesis.json --rpc-http-enabled --rpc-http-api=ETH,NET,IBFT,EEA --host-whitelist=* --rpc-http-cors-origins="all" --privacy-enabled --privacy-url=http://127.0.0.1:8888 --privacy-public-key-file=Orion\nodeKey.pub   
+```
+
+The command line specifies privacy options: 
+
+* [`--privacy-enabled`](../Reference/Pantheon-CLI-Syntax.md#privacy-enabled) enables privacy
+* [`--privacy-url`](../Reference/Pantheon-CLI-Syntax.md#privacy-url) specifies the Orion node URL (`clienturl` in `orion.conf`)
+* [`--privacy-public-key-file`](../Reference/Pantheon-CLI-Syntax.md#privacy-public-key-file) specifies the file containing
+Orion node public key (created in [3. Generate Orion Keys](#3-generate-orion-keys))
+* [`--rpc-http-api`](../Reference/Pantheon-CLI-Syntax.md#rpc-http-api) includes `EEA` enabling privacy JSON-RPC API methods.  
```

Comment:
To run with the default RPC modules along with the eea privacy module you need to specify them explicitly eg `--rpc-http-api=ETH,NET,WEB3,EEA`

Reply:
The command line itself includes all four so I'm not sure I need to re-include the others here when I'm explaining the privacy focused options.  Happy to discuss if you feels it's misleading though.

Reply:
Spoke to Sally and changed the wording to make it clearer that EEA is specified alongside the other options.